packages/aws-cdk/test/commands/diff.test.ts

```diff
@@ -906,7 +906,10 @@ Resources
 
 Stack newGrandChild
 Resources
-[+] AWS::Something SomeResource`);
+[+] AWS::Something SomeResource
+
+Stack UnChangedChild
+There were no differences`);
```

Comment:
this was missed in https://github.com/aws/aws-cdk-cli/pull/264; it should have not been removed, and would have alerted us of the issue.

it also shows an additional thing that this PR fixes. previously, the comment `There were no differences` was printed separately from the diff (on console, it renders the same). now it is included in the formatted diff itself